src/Library/demos/Breakpoints/main.json

```
@@ -0,0 +1,6 @@
+{
+  "category": "layout",
+  "description": "Create adaptive UI elements",
```

I think `elements` might be misleading

```suggestion
  "description": "Create adaptive UI",
```